CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Lower the default `send_attempts` option option to `0`, this disables retries. If you require retries you can increase the `send_attempts` options to the desired value (#1312)
+- Deprecate the `send_attempts` option (#1312)
```

Comment:
Maybe it would be better to squash together these two lines? Otherwise, the user will first read that the default value was changed and then that the option has been deprecated and he may think that the two things are unrelated or he may be confused because it seems that they are in conflict